index.html

```diff
@@ -0,0 +1,152 @@
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/vue_icon.svg">
+				<p>VueJS</p>
+				<p><b>Release: 2014</b></p>
+				<p class="tech_description">VueJS (pronounced: /vju:/, like view) is an open-source progressive JavaScript Framework for building user interfaces.</p>
+			</div>
+		</div>
+
+		<div class="wrapper">
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/elm_icon.svg">
+				<p>Elm</p>
+				<p><b>Release: 2016</b></p>
+				<p class="tech_description">Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces.</p>
+
+			</div>
+
+		</div>
+
+		<div class="wrapper">
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/polymer_icon.svg">
+				<p>Polymer</p>
+				<p><b>Release: 2015</b></p>
+				<p class="tech_description">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+
+			</div>
+
+		</div>
+
+
+	</div>
+
+	<div class="title">
+
+		<img class="img-title" src="assets/triangle.svg">
+		<p class="text-title">Back-End Tools</p>
+
+	</div>
+
+	<div class="card-container">
+
+		<div class="wrapper">
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/rails_icon.png">
+				<p>Ruby on Rails</p>
+				<p><b>Release: 2005</b></p>
+				<p class="tech_description">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+
+			</div>
+
+		</div>
+
+		<div class="wrapper">
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/express_icon.svg">
+				<p>NodeJS</p>
+				<p><b>Release: 2009</b></p>
+				<p class="tech_description">Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.</p>
+
+			</div>
+
+		</div>
+
+		<div class="wrapper">
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/aspnet_icon.png">
+				<p>ASP.NET Core</p>
+				<p><b>Release: 2016</b></p>
+				<p class="tech_description">ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.</p>
+
+			</div>
+
+		</div>
+
+		<div class="wrapper">
+
+			<div class="background-card">
+				<img class="img_tech_logo" src="assets/django_icon.svg">
+				<p>Django</p>
+				<p><b>Release: 2005</b></p>
+				<p class="tech_description">Django is a free and open-source web framework, written in Python, wich follows the model-view-template (MVT) architectural pattern.</p>
+
+			</div>
+
+		</div>
+
+	</div>
+
+	<div class=footer>
+
+		<p class="footer-text">Contact Us!</p>
+		<input type="text" class="footer-input">
+		<button type="button"  class="footer-button">SEND</button>
```

Comment:
This elements should be wrapped in a `form` tag. And I think you missed a little copyright section, check the design :smile: